core/src/main/resources/error/error-classes.json

```diff
@@ -132,7 +132,97 @@
       },
       "INTERCEPTOR_RUNTIME_ERROR" : {
         "message" : [
-          "Error instantiating GRPC interceptor: <msg>"
+          "Error instantiating GRPC interceptor: <msg>."
+        ]
+      },
+      "INVALID_COMMAND_WRITE_INVALID_BUCKET_BY" : {
+        "message" : [
+          "BucketBy must specify a bucket count > 0, received <bucket> instead."
+        ]
+      },
+      "INVALID_PLAN_DEDUPLICATE_INVALID_INPUT" : {
+        "message" : [
+          "Deduplicate must be called with either all or a subset of column names to deduplicate."
+        ]
+      },
+      "INVALID_PLAN_DEDUPLICATE_INVALID_INPUT_COLUMN" : {
+        "message" : [
+          "Invalid column name used for Deduplicate: <col>."
+        ]
+      },
+      "INVALID_PLAN_EMPTY_INPUT" : {
+        "message" : [
+          "Input relation must not be empty."
+        ]
+      },
+      "INVALID_PLAN_INPUT" : {
+        "message" : [
+          "Error transforming Spark Connect plan: <msg>."
+        ]
+      },
+      "INVALID_PLAN_INVALID_ALIAS" : {
+        "message" : [
+          "Alias expressions with more than 1 identifier must not use optional metadata."
+        ]
+      },
+      "INVALID_PLAN_INVALID_DATA_SOURCE" : {
+        "message" : [
+          "Read from DataSource requires a format parameter."
+        ]
+      },
+      "INVALID_PLAN_INVALID_JOIN_CONDITION" : {
+        "message" : [
+          "Using columns or join conditions cannot be set at the same time in Join."
+        ]
+      },
+      "INVALID_PLAN_INVALID_SET_OPERATION" : {
+        "message" : [
+          "Invalid Set operation: <type> does not support <param>."
+        ]
+      },
+      "INVALID_PLAN_TRANSFORM_NA_FILL_EMPTY" : {
```

Comment:
What is NA?

Reply:
It's the underlying operator.

```
 def fillna(
        self,
        value: Union["LiteralType", Dict[str, "LiteralType"]],
        subset: Optional[Union[str, Tuple[str, ...], List[str]]] = None,
    ) -> "DataFrame":
        """Replace null values, alias for ``na.fill()``.
        :func:`DataFrame.fillna` and :func:`DataFrameNaFunctions.fill` are aliases of each other.

```

Reply:
Uhm, I'm now no smarter. The real question is: Will users know what to do with this error?